src/paket.dependencies

```diff
@@ -0,0 +1,7 @@
+source https://api.nuget.org/v3/index.json
+
+nuget FsCheck.Xunit 2.10.6
```

Comment:
please remove the pinned versions from the dependencies file. usually only the lock file has the pins.